app/graphql/mutations/room_create.rb

```diff
@@ -2,7 +2,7 @@
 
 module Mutations
   class RoomCreate < Mutations::BaseMutation
-    argument :name, ID, required: true
+    argument :name, String, required: true
```

Comment:
Noticed this while I was writing some graphql queries into insomnia.  The dope thing about (eventually) switching our specs to use `variables` is that these will be caught automatically!